qucs-doc/tutorial/textmode/Makefile.am

```diff
@@ -31,7 +31,14 @@ TARGETS = textmode.pdf
 
 export TEXINPUTS=$(srcdir):$(srcdir)/..:
 
-tutorial: $(TARGETS)
+textmode.pdf: $(SOURCES)
+
+pics-tutorial:
```

Comment:
probably this should be marked phony, i.e. `.PHONY:  pics-tutorial`.

NB: do you mean tutorial-recursive? iirc newer autotools (will?) support custom recursive targets.